### PR TITLE
Setup script for LANDESK v8 build

### PR DESCRIPTION
**Pull request by howarthm**

### CODE REVIEW

File: ld-v8setup.bat [hunk 1]
```diff
@@ -0,0 +1,38 @@
+echo off
```
Comment (amoravec) [inline ld-v8setup.bat]: do we want to include a copyright header?

File: ld-v8setup.bat [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+echo Get v8 files
+call fetch v8
+
+SET DEPOT_TOOLS_WIN_TOOLCHAIN=0
+SET GYP_MSVS_VERSION=2015
+
+powershell -Command "(Get-Content '%cd%\v8\build\toolchain.gypi') | Foreach-Object {$_ -ireplace '''Optimization''\s*:\s*''\d''', '''Optimization'': ''1'''} | Set-Content  '%cd%\v8\build\toolchain.gypi'"
+powershell -Command "(Get-Content '%cd%\v8\build\toolchain.gypi') | Foreach-Object {$_ -ireplace '''FavorSizeOrSpeed''\s*:\s*''\d''', '''FavorSizeOrSpeed'': ''2'''} | Set-Content  '%cd%\v8\build\toolchain.gypi'"
+REM call %cd%\depot_tools\python276_bin\python .\ld-favorsize.py .\v8\build\toolchain.gypi
```
Comment (amoravec) [inline ld-v8setup.bat]: do we want to check in ld-favorsize.py??  if not, we should just remove this line

File: ld-turnoffwarnings.py [hunk 3]
```diff
@@ -0,0 +1,18 @@
+# Copyright (c) 2016 LANDESK Software Inc. All rights reserved.
+
+""" Turn off warnings as errors when building for Visual Studio 2015 """
+import sys,glob,os
+
+from xml.dom.minidom import parse, parseString
+
+def main(argv):
```
Comment (amoravec) [inline ld-turnoffwarnings.py]: This is way more expressive than trying to do Xml in powershell!

File: ld-v8setup.bat (hunk 1 above)
Comment (howarthm) [inline ld-v8setup.bat]: Good idea.  I added the copyright.

File: ld-turnoffwarnings.py (hunk 3 above)
Comment (howarthm) [inline ld-turnoffwarnings.py]: Sounds like it's OK.  :smile:

**Pull request merged**